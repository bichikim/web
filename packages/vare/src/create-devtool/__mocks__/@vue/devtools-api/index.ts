export const setupDevtoolsPlugin = (options: any, setup: any) => {
  const api = {
    addInspector: (options) => {
      // empty
    },

    on: {
      editInspectorState: (handler: any) => {
        // empty
      },
      getInspectorState: (handler: any) => {
        // empty
      },
      getInspectorTree: (handler: any) => {
        // empty
      },
    },
  }

  setup(api)
}
