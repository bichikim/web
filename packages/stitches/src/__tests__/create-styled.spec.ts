import {createStyled} from '../'
import {renderToString} from '@vue/server-renderer'
import {createSSRApp, h} from 'vue-demi'

describe('create-styled', () => {
  it('should reload', async () => {
    const {styled, toString} = createStyled({
      theme: {},
    })

    const Component = styled('div')

    const app = createSSRApp({
      setup() {
        return () => (
          h(Component)
        )
      },
    })

    await renderToString(app)

    expect(toString()).toMatchSnapshot()
  })
})
