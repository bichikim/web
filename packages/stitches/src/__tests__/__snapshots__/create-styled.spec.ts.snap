// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create-styled should reload 1`] = `""`;
